assets/airports/essa.json

```diff
@@ -0,0 +1,2865 @@
+    },
+    "_comment": "Source restricted areas map: https://www.aro.lfv.se/Editorial/View/8319/ES_AD_2_ESSA_4-1_en",
+    "_comment": "Source restricted areas definition: https://www.aro.lfv.se/Editorial/View/8415/ES_ENR_5_1_en",
+    "_comment": "Most of these areas are only active when military exercies or operations are in progress, so a reasonable selection is implemented",
+    "restricted": [
+        {
+            "name": "R15A",
+            "height": "40500ft",
+            "coordinates": [
+                ["N60d08m16.00", "E018d50m39.00"],
+                ["N60d08m26.14", "E018d51m01.48"],
+                ["N60d08m29.11", "E018d52m16.63"],
+                ["N60d08m30.10", "E018d53m32.00"],
+                ["N60d08m29.11", "E018d54m47.36"],
+                ["N60d08m26.14", "E018d56m02.51"],
+                ["N60d08m21.21", "E018d57m17.24"],
+                ["N60d08m14.32", "E018d58m31.34"],
+                ["N60d08m05.49", "E018d59m44.61"],
+                ["N60d07m54.75", "E019d00m56.84"],
+                ["N60d07m42.13", "E019d02m07.82"],
+                ["N60d07m27.66", "E019d03m17.37"],
+                ["N60d07m11.39", "E019d04m25.29"],
+                ["N60d06m53.35", "E019d05m31.39"],
+                ["N60d06m33.61", "E019d06m35.49"],
+                ["N60d06m12.21", "E019d07m37.41"],
+                ["N60d05m49.21", "E019d08m36.97"],
+                ["N60d05m24.68", "E019d09m34.02"],
+                ["N60d04m58.69", "E019d10m28.40"],
+                ["N60d04m31.31", "E019d11m19.95"],
+                ["N60d04m02.61", "E019d12m08.55"],
+                ["N60d03m32.67", "E019d12m54.05"],
+                ["N60d03m01.58", "E019d13m36.32"],
+                ["N60d02m29.43", "E019d14m15.27"],
+                ["N60d01m56.29", "E019d14m50.78"],
+                ["N60d01m22.27", "E019d15m22.75"],
+                ["N60d00m47.46", "E019d15m51.11"],
+                ["N60d00m11.95", "E019d16m15.77"],
+                ["N59d59m35.84", "E019d16m36.67"],
+                ["N59d58m59.23", "E019d16m53.76"],
+                ["N59d58m22.23", "E019d17m07.00"],
+                ["N59d57m44.92", "E019d17m16.34"],
+                ["N59d57m07.43", "E019d17m21.78"],
+                ["N59d56m29.84", "E019d17m23.29"],
+                ["N59d55m52.26", "E019d17m20.88"],
+                ["N59d55m14.80", "E019d17m14.56"],
+                ["N59d54m37.55", "E019d17m04.34"],
+                ["N59d54m00.63", "E019d16m50.27"],
+                ["N59d53m24.12", "E019d16m32.37"],
+                ["N59d52m48.14", "E019d16m10.72"],
+                ["N59d52m12.77", "E019d15m45.36"],
+                ["N59d51m38.11", "E019d15m16.36"],
+                ["N59d51m04.27", "E019d14m43.82"],
+                ["N59d50m31.32", "E019d14m07.83"],
+                ["N59d50m38.00", "E019d13m56.00"],
+                ["N59d56m32.00", "E018d53m32.00"],
+                ["N60d08m16.00", "E018d50m39.00"]
```

Comment:
Do I need to close the polygon with the starting point? Same applies to all restricted areas.

Reply:
I looked this up myself and have removed the repetition of the first coordinate. Will push soon.